tests/conftest.py

```
@@ -573,3 +583,60 @@ def autouse_fixtures(
+
+@pytest.fixture(scope="class")
+def mariadb_operator_cr(
+    admin_client: DynamicClient, installed_mariadb_operator: None
+) -> Generator[MariadbOperator, Any, Any]:
+    mariadb_csv: ClusterServiceVersion = get_cluster_service_version(
+        client=admin_client, prefix="mariadb", namespace=OPENSHIFT_OPERATORS
+    )
+    alm_examples: list[dict[str, Any]] = mariadb_csv.get_alm_examples()
+    mariadb_operator_cr_dict: dict[str, Any] = next(
+        example for example in alm_examples if example["kind"] == "MariadbOperator"
+    )
```

Pylint (3.3.7)

[refactor] Do not raise StopIteration in generator, use return statement instead

(R1708)